core/src/main/java/io/aiven/klaw/service/KafkaConnectControllerService.java

```diff
@@ -711,6 +712,14 @@ public ApiResponse approveConnectorRequests(String connectorId)
       if (Objects.equals(updateConnectorReqStatus, ApiResultStatus.SUCCESS.value)) {
         setConnectorHistory(connectorRequest, userDetails, tenantId);
```

Comment:
I would consider putting the seConnectorHistory and InsertIntoActivityLog (for all of the variations kafka/connect/schema/acl) into their own methods as its a nice compartmentalized piece of code and with a good method name will be self explanatory about what is happening with a glance in this method.